client/src/commons/DatePick/DateRangePick.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+
+import DatePick from './DatePick'
+
+const DateRangePick: React.FC<Props> = (props: Props): JSX.Element => {
+
+    const { startDate, onStartDateChange, endDate, onEndDateChange, minDate, maxDate, ...rest } = props;
+
+    return (
+        <Grid container direction={'row'} alignItems={'center'} xs={12} spacing={1}>
```

Comment:
dont need curly braces (`direction and alignItems`)

Reply:
👍🏼